package io.zestic.core.exception;

public class RecordNotSavedException extends ApplicationException {

}
